libs/execution_context/ebpf_native.c

```diff
@@ -495,6 +532,11 @@ _ebpf_native_provider_attach_client_callback(
     table->version(&client_context->version);
     if (_ebpf_compare_versions(&client_context->version, &_ebpf_minimum_version) < 0) {
         result = EBPF_INVALID_ARGUMENT;
+        EBPF_LOG_MESSAGE_GUID(
+            EBPF_TRACELOG_LEVEL_ERROR,
+            EBPF_TRACELOG_KEYWORD_NATIVE,
+            "Native module version is older than _ebpf_minimum_version. Rejecting the attach request.",
+            client_module_id);
```

Comment:
Can we get rid of the code below that handles the old contract with out the map_initial_values etc.

Reply:
Done.